specification/applicationinsights/data-plane/Monitor.Exporters/preview/2020-09-15/swagger.json

```diff
@@ -0,0 +1,1044 @@
+          "type": "object",
+          "description": "Stack frame information.",
+          "required": [
+              "level",
+              "method"
+          ],
+          "properties": {
+              "level": {
+                  "type": "number",
+                  "format": "int32",
+                  "description": ""
+              },
+              "method":  {
+                  "type": "string",
+                  "description": "Method name.",
+                  "maxLength": 1024
+              },
+              "assembly":  {
+                  "type": "string",
+                  "description": "Name of the assembly (dll, jar, etc.) containing this function.",
+                  "maxLength": 1024
+              },
+              "fileName":  {
+                  "type": "string",
+                  "description": "File name or URL of the method implementation.",
+                  "maxLength": 1024
+              },
+              "line": {
+                  "type": "number",
+                  "format": "int32",
+                  "description": "Line number of the code implementation."
+              }
+          }
+      },
+      "TrackResponse": {
+          "description": "Response containing the status of each telemetry item.",
+          "type": "object",
+          "properties": {
+              "itemsReceived": {
+                  "type": "number",
+                  "format": "int32",
+                  "description": "The number of items received."
+              },
+              "itemsAccepted": {
+                  "type": "number",
+                  "format": "int32",
+                  "description": "The number of items accepted."
+              },
+              "errors": {
+                  "type":"array",
+                  "description": "An array of error detail objects.",
+                  "items": {
+                      "$ref": "#/definitions/ErrorDetails"
+                  }
+              }
+          }
+      },
+      "ErrorDetails": {
+          "x-ms-client-name": "TelemetryErrorDetails",
+          "description": "The error details",
+          "type": "object",
+          "properties": {
+              "index": {
+                  "type": "number",
+                  "format": "int32",
+                  "description": "The index in the original payload of the item."
+              },
+              "statusCode": {
+                  "type": "number",
+                  "format": "int32",
+                  "description": "The item specific [HTTP Response status code](#Response Status Codes)."
+              },
+              "message": {
+                  "type": "string",
+                  "description": "The error message."
+              }
+          }
+      },
+      "TelemetryEnvelope": {
+          "x-ms-client-name": "TelemetryItem",
+          "description": "System variables for a telemetry item.",
+          "type": "object",
+          "required": [
```

Comment:
kinda surprised `iKey` and `ver` aren't required

Reply:
This is as per the bond specification - https://github.com/microsoft/ApplicationInsights-Home/blob/master/EndpointSpecs/Schemas/Bond/Envelope.bond